Makefile

```diff
@@ -17,10 +22,42 @@ COMMON_ARGS += --frontend-opt build-arg:GOLANG_VERSION=$(GOLANG_VERSION)
 COMMON_ARGS += --frontend-opt build-arg:SHA=$(SHA)
 COMMON_ARGS += --frontend-opt build-arg:TAG=$(TAG)
 
-all: kernel initramfs rootfs osctl-linux-amd64 osctl-darwin-amd64 test lint docs installer
+all: buildkitd builddeps kernel initramfs rootfs osctl-linux-amd64 osctl-darwin-amd64 test lint docs installer
+
+VPATH = $(PATH)
+
+.PHONY: builddeps
+builddeps: gitmeta buildctl
+
+gitmeta:
+	GO111MODULE=off go get github.com/autonomy/gitmeta
+
+buildctl:
+	go get github.com/moby/buildkit/cmd/buildctl
```

Comment:
Can we add in the `GO111MODULE=off` here too? The `go get` command does something really funky and tries to add the deps of any project you `go get` to the `go.mod` and `go.sum` files. Turning it off here will prevent it from mucking with those files.